api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -72,8 +72,7 @@ public ResponseEntity<CriteriaListResponse> findCriteriaAutoComplete(
       String domain,
       String term,
       String type,
-      Boolean standard,
-      Integer limit) {
```

Comment:
Removing limit arg since it's never used by the UI and we default it on the server side.

Reply:
:-)

Reply:
hope the workbench-api yaml also changed...I still see `standard` is still in the query parameter?

Reply:
We still need standard arg.